### PR TITLE
Add preset Cafeteria

### PR DESCRIPTION
**Pull request by Morion-Self**
Add preset for Cafeteria.

See also #25

### CODE REVIEW

File: data/presets/amenity/fast_food/cafeteria.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "icon": "maki-restaurant",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "eat",
+        "breakfast",
+        "dining",
+        "dinner"
```
Comment (tyrasd) [inline data/presets/amenity/fast_food/cafeteria.json:11]: I would replace these generic terms with some that are specific to the preset.

```suggestion
        "canteen",
        "dining hall",
        "lunchroom",
        "refectory"
```

File: data/presets/amenity/fast_food/cafeteria.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    ],
+    "tags": {
+        "amenity": "fast_food",
+        "fast_food": "cafeteria"
+    },
```
Comment (tyrasd) [inline data/presets/amenity/fast_food/cafeteria.json:16]: I'd include the `access` field as some cafeteria's might be not be open to the general public.

```suggestion
    },
    "fields": [
        "{amenity/fast_food}",
        "access_simple"
    ],
```
what do you think?

**Review by tyrasd — COMMENTED** [review]
It's a bit unfortunate that there is also a second tag for these ([amenity=canteen`](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcanteen)). This tag has significantly more uses, though, so I'd be fine to go with this one.

See some comments below.

File: data/presets/amenity/fast_food/cafeteria.json (hunk 2 above)
Comment (Morion-Self) [reply data/presets/amenity/fast_food/cafeteria.json:16]: Yes, I agree

File: data/presets/amenity/fast_food/cafeteria.json (hunk 1 above)
Comment (Morion-Self) [reply data/presets/amenity/fast_food/cafeteria.json]: Yes, I agree

**Review by tyrasd — APPROVED** [review]

**Pull request merged**